### DISCUSSION TITLE
贡献/提问前请看这里

### DISCUSSION THREAD

**Discussion by NeverBehave** [DIYgod/RSSHub#9944] [General]
目前项目核心贡献成员精力不足, 故项目将暂时执行以下策略. 策略将会实行直到本Issue被关闭

**如果的确需要项目重视一些问题, 请给我们反馈! 在对应的issue下增加👍表情并 @NeverBehave 帮助我们判断工作重心.**

**请尽量关注项目Telegram频道，GitHub公告不方便及时通知一些情况**

对本策略的讨论请在下方留言

## 如果你在做贡献

目前我们在issue区为所有确定方向的issue打上了 [Contribution Welcome](https://github.com/DIYgod/RSSHub/issues?q=is%3Aissue+is%3Aopen+label%3A%22contribution+welcome%22) 标签, 如果可以请优先考虑这些Issue的贡献

### 核心问题

目前项目发展到现在, 部分问题比如 #6198 等需要核心开发者以及社区的帮助. 

请优先查看 [High Priority](https://github.com/DIYgod/RSSHub/issues?q=label%3A%22high+priority%22+), [Help wanted](https://github.com/DIYgod/RSSHub/issues?q=label%3A%22help+wanted%22) 标签下的问题

一般用户: [Discussion](https://github.com/DIYgod/RSSHub/issues?q=label%3Adiscussion) 标签下的问题一些反馈可以帮助我们定位项目未来方向 - 欢迎你的加入!

### 关于新路由

我们仍会持续追踪跟进新的路由PR, 目前我们在一套自动测试PR路由的工作流 - 如果遇到问题欢迎反馈!

### 如果是关于新的RSS提案

除非有成员愿意主动贡献, 否则这些Issue在本策略下将不会主动处理 - 低优先级

建议有动手能力的用户可以尝试一下贡献规则, 如果遇到问题, 可以在 [留言板](https://github.com/DIYgod/RSSHub/discussions/categories/q-a) 开启新讨论, **这些问题可以帮助我们完善贡献指南**

不要担心问题太蠢 - 也许你的疑惑正是文档所缺失的

一般用户: 你可以使用Issuehunt添加赏金: 这能够大大增加Issue被解决的概率, 参考指南: https://github.com/DIYgod/RSSHub/discussions/6693

### 如果是关于已有RSS的问题

我们会尽量保证路由可用, 但这主要依赖之前的维护者以及社区需求 - 请给我们反馈! (Thumbs up)

**Comment by NeverBehave** [comment]
https://github.com/DIYgod/RSSHub/issues/7201

**Comment by NeverBehave** [comment]
#6795 在pubDate问题处理完成之前暂时停止新路由合并

**Comment by Leozgu** [comment]
你好 我在自己服务器使用的docker 部署的rsshub。
因为订阅时 已有的 RSSHub 规则显示有些问题  我便想修改一下 RSSHub    （规则.js）文件
我用 docker exec  命令 进入容器 修改好 退出 再用 docker commit 保存了一份镜像  
但是我启用这个新镜像的时候发现会自动退出 估计是出问题了
请问有什么方法可以 增加或者修改 RSSHub 规则吗

**Comment by NeverBehave** [comment]
pubDate 问题：https://docs.rsshub.app/joinus/pub-date.html 暂时解决
cache 解耦提案预备中

**Comment by NeverBehave** [comment]
https://github.com/DIYgod/RSSHub/discussions/7740 cache动议

**Comment by Colin-XKL** [comment]
> 你好 我在自己服务器使用的docker 部署的rsshub。
> 因为订阅时 已有的 RSSHub 规则显示有些问题 我便想修改一下 RSSHub （规则.js）文件
> 我用 docker exec 命令 进入容器 修改好 退出 再用 docker commit 保存了一份镜像
> 但是我启用这个新镜像的时候发现会自动退出 估计是出问题了
> 请问有什么方法可以 增加或者修改 RSSHub 规则吗

既然你会修改RSSHub的规则文件，那想必是有一定开发能力的，可以考虑下在RSSHub源码的基础上更改，测试没问题后，将更改提交至RSSHub主分支，后面你的更改被合并之后，你就可以直接使用官方的RSSHub镜像啦，而且你也为RSSHub的社区做了贡献

**Comment by NeverBehave** [comment]
所有feat PR暂缓合并：route.js 需要更好的管理方式

**Comment by zreo0** [comment]
请问下关于新管理方式的进展如何了？

**Comment by NeverBehave** [comment]
我们正在尝试推行新的插件规范，进度：https://github.com/DIYgod/RSSHub/pull/8224/

**Comment by aidistan** [comment]
> 我们正在尝试推行新的插件规范，进度：#8224

除了一方插件，会考虑三方插件模式吗？目前通过 PR 更新 route 的办法耗费了太多维护者的精力...

**Comment by NeverBehave** [comment]
> > 我们正在尝试推行新的插件规范，进度：#8224
> 
> 除了一方插件，会考虑三方插件模式吗？目前通过 PR 更新 route 的办法耗费了太多维护者的精力...

这是我正在尝试迁移的方向，我还在考虑做法
但是我一个人考虑这些兼容必然有不周到的地方，目前v2是我强推想打破这个局面的一个方式……希望可以看到更多参与进核心开发的成员

**Comment by aidistan** [comment]
> 这是我正在尝试迁移的方向，我还在考虑做法
> 但是我一个人考虑这些兼容必然有不周到的地方，目前v2是我强推想打破这个局面的一个方式……希望可以看到更多参与进核心开发的成员

辛苦了，从 v2 路由规范里能看出来这个方向，但感觉缺少参与到核心开发的阶梯。目前由核心开发者在 Discussions 中提出动议、外围贡献者参与讨论进而贡献的方式很好，期待开展该方向的相关讨论/动议~

**Comment by NeverBehave** [comment]
目前我们集中的方向是

1. 迁移到ESM
2. 自动化ISSUE流程
3. 插件文档规范定义

**Comment by NeverBehave** [comment]
https://github.com/DIYgod/RSSHub/pull/8337 ESM

**Comment by NeverBehave** [comment]
目前无暇关注更新，有PR审核需求请 @DIYgod

**Comment by NeverBehave** [comment]
https://github.com/DIYgod/RSSHub/pull/9758 测试新的issue分类+通知规则
